05LessonFive/Checkpoint1/css/style.css

```diff
@@ -1,43 +1,42 @@
-.lawn {
-    width: 100%;
-    height: 300px;
+.wrapper {
+    width:1000px;
+    margin:0 auto;
 }
 
-.lawn h1 {
-    text-align: center;
-    font-family: monospace;
-    color: darkgray;
-}
-
-.lawn {
-    background-image: url(../img/banner-garden-stmichaelsmount.jpg);
-    background-repeat: no-repeat;
-    background-position: center;
-    background-size: cover;
+.box1 {
+    width:500px;
+    float:left;
+    margin-left: 15px;
 }
 
-.left {
-    float: left;
-    width: 60%;
-    height: 300px;
-    margin-left: 260px;
-    background-color: lightgray;
+.img {
+    width:500px;
 }
 
```

Comment:
hint: if you set the container to a specific width and then your images' width to 100%, they'll fill the space!